test/unitTests/OmnisharpManager.test.ts

```diff
@@ -136,7 +136,7 @@ suite(OmnisharpManager.name, () => {
             });
 
             test('Returns the default path if the omnisharp path is not set', async () => {
-                let launchInfo = await manager.GetOmniSharpLaunchInfo(defaultVersion, "", useFramework, server.baseUrl, latestfilePath, installPath, extensionPath);
+                const launchInfo = await manager.GetOmniSharpLaunchInfo(defaultVersion, undefined, useFramework, server.baseUrl, latestfilePath, installPath, extensionPath);
```

Comment:
I think we should add an additional test here for cases when is set to empty string. Users are very likely to leave `omnisharp.path: ""` in their settings.json when trying to switch from "latest" back to release and the empty string case should be treated like `undefined`.